docs/src/developer/guide/architecture.md

```diff
@@ -1,10 +1,42 @@
 # System Architecture
-<!-- Replace all of the titles with relevant titles -->
+In this section, we outline the system architecture using ER Diagrams, Software Component Diagrams etc. and key libraries / frameworks used in this project.
 
 ## Frameworks Used
```

Comment:
Please refer to the [documentation naming conventions](https://kartoza.github.io/sawps/developer/documentation/#file-naming-conventions) for heading conventions.